tests/e2e/test_rosacli_machine_pool.go

```diff
@@ -155,3 +158,85 @@ var _ = Describe("Edit machinepool",
 
 			})
 	})
+
+var _ = Describe("Edit machinepool",
+	labels.Day2,
+	labels.FeatureMachinepool,
+	labels.NonHCPCluster,
+	func() {
+		defer GinkgoRecover()
+		var (
+			clusterID          string
+			rosaClient         *rosacli.Client
+			machinePoolService rosacli.MachinePoolService
+			isHosted           bool
+		)
+
+		BeforeEach(func() {
+			By("Get the cluster")
+			clusterID = config.GetClusterID()
+			Expect(clusterID).ToNot(Equal(""), "ClusterID is required. Please export CLUSTER_ID")
+
+			By("Init the client")
+			rosaClient = rosacli.NewClient()
+			machinePoolService = rosaClient.MachinePool
+
+			var err error
+			isHosted, err = rosaClient.Cluster.IsHostedCPCluster(clusterID)
+			Expect(err).ToNot(HaveOccurred())
+
+			if isHosted {
+				Skip("This test case only work for classic cluster")
+				return
```

Comment:
Is it required "return" ?

Reply:
@yingzhanredhat Good catch. Removed since we have `labels.NonHCPCluster,` now.